andela/banshee-bn-backend
# #168189966 Fix duplicate validation implementation

#### What does this PR do?
Have duplicate validation middleware removed and refactor login schema
#### Description of Task to be completed?
- Refactor login schema
- Refactor login route handler
- Add more mock data for login
- Modify login test
- Modify server entrypoint - not to use morgan in test environment

#### How should this be manually tested?
1. Clone the repository
2. Cd into the repository
3. Run `npm install`
4. Create `.env` file and declare all your variables for database and others
5. Make sure Postgres database is installed
6. Create database in Postgres
7. Run `npm test`

#### What are the relevant pivotal tracker stories?
[#168189966](https://www.pivotaltracker.com/story/show/168189966)
#### Any background context you want to add?
Nil
#### Screenshots
Nil
